deploy/crds/chaosengine_crd.yaml

```diff
@@ -171,12 +171,1828 @@ spec:
                                         type: string
                                   source:
                                     type: object
+                                    description: The source where we have to run the
+                                      command It will run in inline(inside experiment
+                                      itself) mode if source is nil
                                     properties:
+                                      args:
+                                        description: Args of runner
```

Comment:
Can we add a more detailed description here? Also it should be Args of the source pod

Reply:
Please check the other places too